www/js/diary/infinite_scroll_list.js

```diff
@@ -267,30 +290,40 @@ angular.module('emission.main.diary.infscrolllist',['ui-leaflet',
       f.state = false;
     });
     $scope.allTrips = true;
-    $scope.recomputeDisplayTrips();
+    $scope.recomputeDisplayTimelineEntries();
     $ionicScrollDelegate.scrollBottom();
     ClientStats.addReading(ClientStats.getStatKeys().LABEL_TAB_SWITCH, {"source": prev, "dest": $scope.getActiveFilters()});
   }
 
-  $scope.recomputeDisplayTrips = function() {
+  $scope.recomputeDisplayTimelineEntries = function() {
     console.log("recomputing display trips now");
     let alreadyFiltered = false;
     $scope.filterInputs.forEach((f) => {
         if (f.state == true) {
             if (alreadyFiltered) {
                 Logger.displayError("multiple filters not supported!", undefined);
             } else {
-                // console.log("Trip n before: "+$scope.data.displayTrips.length);
+                // console.log("Trip n before: "+$scope.data.displayTimelineEntries.length);
                 $scope.data.displayTrips = $scope.data.allTrips.filter(
                     t => (t.waitingForMod == true) || f.filter(t));
-                // console.log("Trip n after:  "+$scope.data.displayTrips.length);
+                // console.log("Trip n after:  "+$scope.data.displayTimelineEntries.length);
                 alreadyFiltered = true;
             }
         }
     });
     if (!alreadyFiltered) {
         $scope.data.displayTrips = $scope.data.allTrips;
     };
+    
+    $scope.data.displayTimelineEntries = []
+    $scope.data.displayTrips.forEach((t) => {
+      const place = t.confirmed_place;
+      const {confirmed_place, ...trip} = t;
+      $scope.data.displayTimelineEntries.push(trip);
+      if ($scope.showPlaces && place) {
+        $scope.data.displayTimelineEntries.push(place);
+      }
+    });
```

Comment:
Putting in this comment for reference: we may want to investigate this for the issue where "after labeling trips with enketo survey and then adding trip/place addition notes -> then going on another trip -> then running the pipeline -> app becomes hung up and unresponsive"